lighthouse-core/audits/estimated-input-latency.js

```diff
@@ -45,23 +46,36 @@ class EstimatedInputLatency extends Audit {
       throw new LHError(LHError.errors.NO_FMP);
     }
 
-    const latencyPercentiles = TracingProcessor.getRiskToResponsiveness(tabTrace, startTime);
-    const ninetieth = latencyPercentiles.find(result => result.percentile === 0.9);
-    const rawValue = parseFloat(ninetieth.time.toFixed(1));
+    const events = TracingProcessor.getMainThreadTopLevelEvents(tabTrace, startTime)
+      .filter(evt => evt.duration >= 1);
+
+    const candidateStartEvts = events.filter(evt => evt.duration >= 10);
```

Comment:
similarly, can we raise 10 to something like 50?  Seems like we'd lose precision in the "you're good" area of scores, but that doesn't seem too bad.

Reply:
yeah sgtm 👍 much faster too

Reply:
doesn't that make the max score the PODR?

Reply:
oh, only filtering the start event times